kao.sty

```diff
@@ -609,7 +609,7 @@
 	\marginskip{\kaomarginskipabove}%
 	\begin{lrbox}{\@sidenotes@marginlistingbox}%
 	\begin{minipage}{\marginparwidth}%
-		\captionsetup{type=lstlisting}%
+		\captionsetup{type=listing}%
```

Comment:
It seems to me that unifying the counters for the `lstlisting` and `listing` environments is very difficult. And I cannot statically pick one of them here. Now I think I should rather introduce a new option to let the user pick `listings` or `minted` (and keep `\captionsetup*`), which requires some more refactoring work...